tests/test_models.py

```diff
@@ -272,6 +272,36 @@ def test_get_course_enrollment_url_no_site_config(self):
         with raises(NotConnectedToOpenEdX, message=error):
             customer.get_course_enrollment_url('course_id')
 
+    @ddt.data(
+        ('course_exists', True),
+        ('fake_course', False),
+        ('course_also_exists', True)
+    )
+    @ddt.unpack
+    @mock.patch('enterprise.models.CourseCatalogApiClient')
+    def test_course_in_catalog(self, received_course_id, expected_result, mock_catalog_api_class):
```

Comment:
@haikuginger Why should we call the first parameter `received_course_id` instead of just `course_id`?

**EDIT**: I get it now; when `catalog_contains_course_run` is called, `course_id` that gets passed to it comes from request data. However, I don't think this matters in the context of this test (and in theory there's nothing stopping us from calling this method with data that doesn't come from a request).